Add Projects page

jholdstock opened the pull request:
Draft PR so the general layout of the page can be reviewed.

jholdstock commented:
Requesting a review on this. It's not complete but I think its a good enough starting point. Opened #5 to flesh this out further in future.

s-ben commented on docs/getting_started/projects.md:
```diff
@@ -1,4 +1,176 @@
+- [decred/dcrwallet](https://github.com/decred/dcrwallet) (Go)
+
+dcrwallet is the daemon that handles Decred wallet functionality for a single user.
+It manages all of your accounts, addresses, and transactions;
```
Maybe use commas instead of `;`s here? Reads fine actually with ; but trips me up slightly

jholdstock replied:
Yeah I thought the semi-colon use was strange too tbh. I copied it from https://docs.decred.org/wallets/cli/dcrwallet-setup/

s-ben commented on docs/getting_started/projects.md:
```diff
@@ -1,4 +1,176 @@
+
+## Lightning Network
+
+- [decred/dcrlnd](https://github.com/decred/dcrlnd) (Go)
```
Add a short description for each repo?

s-ben commented on docs/getting_started/projects.md:
```diff
@@ -1,4 +1,176 @@
+
+- [decred/dcrstakepool](https://github.com/decred/dcrstakepool) (Go)
+
+dcrstakepool is the reference implementation for a Decred VSP
```
VSP (Voting Service Provider) -> Voting Service Provider (VSP)

We're not totally consistent on this, but I think we do the latter more often

s-ben commented on docs/getting_started/projects.md:
```diff
@@ -1,4 +1,176 @@
+- [decred/dcrstakepool](https://github.com/decred/dcrstakepool) (Go)
+
+dcrstakepool is the reference implementation for a Decred VSP
+(Voting Service Provider). It consists of two major components,
```
...two major components`,` a ->...two major components`:` a 

Both work I guess

s-ben commented on docs/getting_started/projects.md:
```diff
@@ -1,4 +1,176 @@
+
+- [decred/dcrpool](https://github.com/decred/dcrpool) (Go)
+
+dcrpool is a lightweight proof-of-work mining pool which operates without collecting
```
proof-of-work -> Proof-of-Work (PoW)

jholdstock commented:
Thanks for the review. I have addressed all of the comments apart from [the one about the formatting](https://github.com/jholdstock/dcrdevdocs/pull/2#discussion_r319349141). Copying that comment into #5
